addon-test-support/type-text.js

```diff
@@ -0,0 +1,36 @@
+import { run } from '@ember/runloop';
+import getElementWithAssert from './-private/get-element-with-assert';
+import { focus } from './focus';
+import { fireEvent } from './fire-event';
+import { keyEvent } from './key-event';
+import wait from 'ember-test-helpers/wait';
+
+const BACKSPACE_CODE = 8;
+/*
+  @method typeText
+  @param {String|HTMLElement} selector
+  @param {String} text
+  @return {RSVP.Promise}
+  @public
+*/
+export function typeText(selector, text) {
+  let el = getElementWithAssert(selector);
+
+  run(() => focus(el));
+  keyEvent(el, 'keydown', BACKSPACE_CODE);
+  run(() => el.value = '');
+  keyEvent(el, 'input');
```

Comment:
`input` is not a key event, that one must be fired with `run(() => fireEvent(el, 'input'));`

I'm also thinking that probably I wouldn't bake in the backspace simulation in the helper. I think that user could start typing right away.

Reply:
Oh boy, that's what you get when you try to get something done on friday evening... : ( Will be done shortly.

Reply:
Actually, how should removing current input be handled? There might be already something in the input, and  helper probably should overwrite it, not append to it. Should it simply clean the value, but  not fire any events?

Reply:
@cibernox I suppose I should have mentioned you.